src/js/media-theme-element.js

```diff
@@ -7,7 +7,9 @@ import { camelCase } from './utils/utils.js';
 export * from './utils/template-parts.js';
 
 const observedMediaAttributes = {
+  'media-target-live-window': 'targetLiveWindow',
```

Comment:
Is this just a placeholder for a future media state?

Reply:
yes, I can set it manually in Mux player for the time being